pkgs/top-level/all-packages.nix

```diff
@@ -4635,7 +4635,7 @@ with pkgs;
 
   circus = callPackage ../tools/networking/circus { };
 
-  citrix_workspace = citrix_workspace_21_12_0;
+  citrix_workspace = citrix_workspace_22_05_0;
 
   inherit (callPackage ../applications/networking/remote/citrix-workspace { })
     citrix_workspace_20_04_0
```

Comment:
lets be honest: do we really need **every** version here?

Reply:
You're right of course. I have dropped support for the 20.X releases.